_conferences/droidcon-cluj-2019.md

```diff
@@ -0,0 +1,11 @@
+---
+name: "Droidcon Transylvania"
```

Comment:
We don't repeat the location in the name because the location is listed directly next to it
```suggestion
name: "Droidcon"
```

Reply:
It was more part of the branding than the actual location, but I see your point. Better to keep it simple.